### PR TITLE
RedHat Cheat sheet: Adding alias

### PR DESCRIPTION
**Pull request by ankushg07**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`{RedHat Cheatsheet}: {Adding alias for the IA}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes
Added couple of alias 

What changes does this PR introduce? 
Cheat sheet can be triggered by alias also

Instant Answer Page: https://duck.co/ia/view/redhat_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@ankushg07 Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/redhat.json [hunk 1]
```diff
@@ -7,8 +7,15 @@
         "sourceUrl": "https://access.redhat.com/articles/yum-cheat-sheet"
     },
     "aliases": [
+<<<<<<< HEAD
```
Comment (amarlearning) [inline share/goodie/cheat_sheets/json/redhat.json]: remove this `<<<<<<< HEAD`

File: share/goodie/cheat_sheets/json/redhat.json [hunk 2]
```diff
@@ -7,8 +7,15 @@
...
+        "Red hat",
+        "yum commands"
+        
+      ],
+=======
```
Comment (amarlearning) [inline share/goodie/cheat_sheets/json/redhat.json]: and this `=======`

File: share/goodie/cheat_sheets/json/redhat.json (hunk 1 above)
Comment (amarlearning) [inline share/goodie/cheat_sheets/json/redhat.json]: remove this `<<<<<<< HEAD`

File: share/goodie/cheat_sheets/json/redhat.json [hunk 3]
```diff
@@ -7,8 +7,15 @@
         "sourceUrl": "https://access.redhat.com/articles/yum-cheat-sheet"
     },
     "aliases": [
+<<<<<<< HEAD
+        "Red hat",
```
Comment (amarlearning) [inline share/goodie/cheat_sheets/json/redhat.json]: and this ``"Red hat",

File: share/goodie/cheat_sheets/json/redhat.json [hunk 4]
```diff
@@ -7,8 +7,15 @@
...
+        "yum commands"
```
Comment (amarlearning) [inline share/goodie/cheat_sheets/json/redhat.json]: and this `"yum commands"`

File: share/goodie/cheat_sheets/json/redhat.json (hunk 2 above)
Comment (amarlearning) [inline share/goodie/cheat_sheets/json/redhat.json]: remove line 14 & 15 as well!

File: share/goodie/cheat_sheets/json/redhat.json [hunk 5]
```diff
@@ -7,8 +7,15 @@
...
+        
+      ],
+=======
         "red hat", "redhat yum", "redhat enterprise yum", "red hat yum", "red hat enterprise yum"
     ],
+>>>>>>> master
```
Comment (amarlearning) [inline share/goodie/cheat_sheets/json/redhat.json]: remove this also `>>>>>>> master`

**Comment by gautamkrishnar** [comment]
@ankushg07 there's no files in this PR

**Comment by ankushg07** [comment]
@gautamkrishnar yeah .. sorry !!
actually my forked Repo was not updated and i pushed it with outdated forked repo

**Comment by gautamkrishnar** [comment]
@ankushg07 can you please update this process or close this one

**Pull request closed without merging**